docs/guides/tms.rst

```diff
@@ -0,0 +1,242 @@
+TMS Servers
+===========
+
+GeoPySpark is meant to work with geospatial data.  The most natural way to
+interact with these data is to display them on a map.  In order to allow for
+this interactive visualization, we provide a means to create Tile Map Service
+(TMS) servers directly from both GeoPySpark RDDs and tile catalogs.  A TMS
+server may be viewed using a web-based tool such as geojson.io_ or interacted
+with using the GeoNotebook_ Jupyter kernel. [#]_
+
+Note that the following examples rely on this common boilerplate code:
+
+.. code-block:: python
+
+   from geopyspark import geopyspark_conf
+   from geopyspark.geotrellis.tms import TMS
```

Comment:
I'm not sure if this is a major deal, but imports are handled differently in the other examples in the documentation.

Instead of

```python
from geopyspark import geopyspark_conf
from geopyspark.geotrellis.tms import TMS
```

it'd be

```python
import geopyspark as gps
```